scripts/build_and_test.sh

```diff
@@ -0,0 +1,10 @@
+#!/usr/bin/env bash
+
+# build
+docker-compose -f docker-compose.yml build static_site
+
+# test
+docker-compose run -u $(id -u ${USER}):$(id -g ${USER}) --publish 4000:4000 --rm --entrypoint "bundle exec jekyll serve -H 0.0.0.0" -d static_site
+sleep 20
+curl localhost:4000 | grep "Join the Google Group"
```

Comment:
A good way to test this is to push something like 
`curl localhost:4000 | grep "Blah blah blah"` to see if Travis blocks.